axelrod/moran.py

```diff
@@ -92,6 +92,8 @@ def __init__(
         noise:
             The background noise, if any. Randomly flips plays with probability
             `noise`.
+        game:
```

Comment:
could we make this
`game: axelrod.Game`
`    The game object used to score the match`
to match other docstrings

Reply:
done